api/lib/data-cube/table/index.ts

```diff
@@ -37,7 +39,10 @@ export const createFactTable = asyncMiddleware(async (req: express.Request, res:
   })
     .commit(projects)
 
-  const tableId = `${projectId}/table/${variables.name.value}`
```

Comment:
This was the problematic part I left out which took a shortcut to the fact table id, which was actually incorrect having been  slugified